.github/workflows/build-and-push.yml

```diff
@@ -0,0 +1,39 @@
+name: Build and push container images to Quay.io registry
+on:
+  push:
+    branches:
+      - master
+  schedule:
+      - cron: '0 1 * * 3'
+
+jobs:
+  build-and-push:
+    if: github.repository_owner == 'sclorg'
+    runs-on: ubuntu-20.04
+    strategy:
+      fail-fast: false
+      matrix:
+        include:
+          - dockerfile: "3.3/Dockerfile.fedora"
+            registry_namespace: "fedora"
+            quayio_username: "QUAY_IMAGE_FEDORA_BUILDER_USERNAME"
+            quayio_token: "QUAY_IMAGE_FEDORA_BUILDER_TOKEN"
+            tag: "33"
+            docker_context: "3.3"
+            image_name: "ruby-33"
+
+    steps:
+      - name: Build and push to quay.io registry
+        uses: sclorg/build-and-push-action@v4
+        with:
+          registry: "quay.io"
+          registry_namespace: ${{ matrix.registry_namespace }}
+          registry_username: ${{ secrets[matrix.quayio_username] }}
+          registry_token: ${{ secrets[matrix.quayio_token] }}
+          dockerfile: ${{ matrix.dockerfile }}
+          docker_context: ${{ matrix.docker_context }}
+          tag: ${{ matrix.tag }}
```

Comment:
wouldn't this result in the full image name of `quayi.io/fedora/ruby-33:33` ? Instead of `tag: "33"`, `tag: "latest"` would probably make more sense, we already know it's Ruby 3.3 from `image_name: "ruby-33"` IMO.

Reply:
The latest tag is added automatically, by our GitHub Action https://github.com/sclorg/build-and-push-action/blob/main/action.yml#L106

Reply:
Ok, interesting.

Then I think this PR is OK from my POV :]